frontend/src/lib/components/PropertiesTable/PropertiesTable.tsx

```diff
@@ -209,12 +215,26 @@ export function PropertiesTable({
     const [searchTerm, setSearchTerm] = useState('')
     const { hidePostHogPropertiesInTable } = useValues(userPreferencesLogic)
     const { setHidePostHogPropertiesInTable } = useActions(userPreferencesLogic)
+    const { isCloudOrDev } = useValues(preflightLogic)
 
     const objectProperties = useMemo(() => {
         if (!properties || Array.isArray(properties)) {
             return []
         }
-        let entries = Object.entries(properties)
+        let entries = Object.entries(properties).sort((a, b) => {
```

Comment:
a little duplication for now so i can get to something shippable before eating :)